extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,739 @@
+
+>A **quadtree** divides space only horizontally. It divides each bounding volume into 4 smaller bounding volumes where each horizontal dimension is halved. The vertical dimension remains the same. The resulting tree has a branching factor of 4.
+
+Horizontal dimensions are `(x, y)` when the bounding volume is a `box` and `(longitude, latitude)` when the bounding volume is a `region`.
+
+![Quadtree](figures/quadtree.png)
+
+>An **octree** divides space along all 3 dimensions. It divides each bounding volume into 8 smaller bounding volumes where each dimension is halved. The resulting tree has a branching factor of 8.
+
+![Octree](figures/octree.png)
+
+The following diagrams illustrate the subdivision in the bounding volume types supported by 3D Tiles:
+
+| Root Box | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![Root box](figures/box.png) | ![Box Quadtree](figures/box-quadtree.png) | ![Box octree](figures/box-octree.png)  |
+
+| Root Region | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![Root region](figures/region.png) | ![Region Quadtree](figures/region-quadtree.png) | ![Region octree](figures/region-octree.png)  |
+
+### Implicit Subdivision
+
+Implicit tiling only requires defining the tiling scheme, bounding volume, geometric error, and refine strategy at the root tile. These properties can be computed for any other tile based on the following rules:
+
+| Property | Subdivision Rule | 
+| --- | --- |
+| `tilingScheme` | Constant for all tiles in tileset |
+| `refine` | Constant for all tiles in tileset |
+| `boundingVolume` | If `tilingScheme` is `QUADTREE`, the parent tile is divided into 4 child tiles. If `tilingScheme` is `OCTREE` the child tile is divided into 8 child tiles. |
+| `geometricError` | Each child's `geometricError` is half of the parent's `geometricError` |
+
+## Tile Coordinates
+
+>**Tile coordinates** are a tuple of integers that uniquely identify a tile. Tile coordinates are either `(level, x, y)` for quadtrees, and `(level, x, y, z)` for octrees.
+
+Tile coordinates are interpreted differently depending on the type of bounding volume (`box` or `region`).
+
+For `box`, the tile coordinates are listed along the same direction as the Cartesian coordinate system of the tileset. That is:
+
+| Coordinate | Description |
+| --- | --- |
+| `level` | The 0-indexed level number. The root tile is at level 0, the root tile's children are at level 1, and so on. |
+| `x` | The 0-indexed number of tiles from the side of the root bounding box in the `+x` direction. This is measured within a single level. |
+| `y` | The 0-indexed number of tiles from the side of the root bounding box in the `+y` direction. This is measured within a single level.|
+| `z` (only for octrees) | The 0-indexed number of tiles from the side of the root bounding box in the `+z` direction. This is measured within a single level. |
```

Comment:
Reword these a little.  should be "number of tiles from the side of the root bounding box in order of increasing x coordinate"

However, if we can find a more concise way to express this, that would be even better.

Reply:
Instead of adding verbosity in the coordinate descriptions, consider adding some text immediately before or after this table noting that all coordinates are 0 indexed.